experimenter/experimenter/nimbus_ui_new/templatetags/nimbus_extras.py

```diff
@@ -41,3 +43,31 @@ def pagination_url(context, page, **kwargs):
         return f"?{data.urlencode()}"
     else:
         return "."
+
+
+@register.filter(name="yesno")
+def yesno(value):
+    if value is None:
+        return ""
+    return "Yes" if value else "No"
```

Comment:
This already exists.

https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#yesno